Resolutions/PacMan_Resolution/rules.txt

```diff
@@ -0,0 +1,130 @@
+[Definition]
+titleIds = 0005000010146300,0005000010147900,000500001014F300,000500001017E600,000500001017E500,0005000010182300
+name = Resolution
+path = "PAC-MAN & the Ghostly Adventures 1 & 2/Graphics/Resolution"
```

Comment:
Is it possible to separate the PAC-MAN pack into two packs? It would be one for GA1 and another for GA2.

Reply:
Alright, I separated them. I wasn't sure whether or not to split them because there are combined graphic packs for games like New Super Mario Bros. U & Luigi U, so I needed some clarification.